cosmosdb/sql/ps-account-failover-priority-update.ps1

```diff
@@ -1,14 +0,0 @@
-# Change the failover priority for an Azure Cosmos Account
```

Comment:
This script is used by \azure-docs-pr\articles\cosmos-db\scripts\powershell\common\ps-account-failover-priority-update.md file

Reply:
Thanks for catching. I have created an updated version of this script. I will modify the samples page to point to the new one after this merges.